engine/pkg/meta/model/client.go

```diff
@@ -11,27 +11,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package extension
+package common
```

Comment:
Why is it still `package common`?

Reply:
Fix it. 
The `make engine` and `make check` can't check this situation.